storage/src/time_stamp_comparator.cpp

```diff
@@ -0,0 +1,70 @@
+#include "rocksdb/time_stamp_comparator.h"
+#include "endianness.hpp"
+#include "assertUtils.hpp"
+
+namespace concord {
+namespace storage {
+namespace rocksdb {
+
+#ifdef USE_ROCKSDB
+
+using namespace ::rocksdb;
+
+class Lexicographic64TsComparator : public Comparator {
+ public:
+  Lexicographic64TsComparator() : Comparator(/*ts_sz=*/sizeof(uint64_t)), cmp_without_ts_(BytewiseComparator()) {
```

Comment:
cmp_without_ts_ is a pointer, should this be initialized with new BytewiseComparator()

Reply:
It's a RocksDB method call defined at ```util/comparator.cc```

Reply:
it returns a pointer

Reply:
thanks @cloudnoize